fb_login.py

```diff
@@ -1,19 +1,25 @@
 from selenium import webdriver
 import getpass
 
-print("please enter your username and password for facebook login")
-username = input("enter you username: ")
-password = getpass.getpass("enter your password it would be hidden: ")
+print("Please enter your details for facebook login")
+# username = input("enter you username: ")
+# password = getpass.getpass("enter your password it would be hidden: ")
 # Enter your password and it's safe and hidden
 
+  # To open the web browser(google chrome)
 
-driver = webdriver.Chrome()  # To open the web browser(google chrome)
 
+class Facebook:
 
-def fb_login(username, password):
-    driver.get('https://www.facebook.com')  # To hit the URL(facebook)
-    driver.find_element_by_id('email').send_keys(username)
-    driver.find_element_by_id('pass').send_keys(password)
-    driver.find_element_by_id('loginbutton').click()
+    def __init__(self,username,password):
+        self.username = input("enter your username: ")
+        self.password = getpass.getpass("enter your password: ")
 
-fb_login(username, password)
+    def fb_login(self):
+        driver = webdriver.Chrome()

```

Comment:
well, initiate driver in the `__init__` method itself and then use here as `self.driver`

this will help us carry the same session across all methods of the Facebook class.